lib/cli-command.js

```diff
@@ -142,7 +142,7 @@ ${postCreateInstructions(createOptions)}`);
     default: 'example',
   }, {
     command: '--example-react-native-version [exampleReactNativeVersion]',
-    description: 'React Native version for the generated example project',
+    description: 'React Native template version for the generated example project, for example: react-native-tvos, react-native-tvos@0.62.2-1 (requires --tvos-enabled option); react-native@0.62, react-native@next',
```

Comment:
```suggestion
    description: 'React Native template version for the generated example project, for example: react-native-tvos, react-native-tvos@0.62.2-1 (requires --tvos-enabled option); react-native@0.62',
```

(reverting a recent change)

as using react-native@next may trigger issue #422